service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationController.java

```diff
@@ -43,4 +50,15 @@ public ResponseEntity createdCase(
 
         return ResponseEntity.ok(body);
     }
+
+    @PostMapping("/submitted")
+    public ResponseEntity handleCaseInitiation(
+        @RequestHeader(value = "authorization") String authorization,
+        @RequestHeader(value = "user-id") String userId,
+        @RequestBody CallbackRequest callbackRequest) {
+
+        applicationEventPublisher.publishEvent(new InitiatedCaseEvent(callbackRequest, authorization, userId));
+
+        return new ResponseEntity(HttpStatus.OK);
```

Comment:
Please remove and change handleCaseInitiation to return void. It will work exactly same way.